### PR TITLE
Apply black style to test_m* files, version 19.10b0.

### PR DESCRIPTION
**Pull request by svalqui**
This pull request addresses issue #2552 

- [X] I hereby agree to dual licence this and any previous contributions under both
the _Biopython License Agreement_ **AND** the _BSD 3-Clause License_.

- [X] I have read the ``CONTRIBUTING.rst`` file, have run ``flake8`` locally, and
understand that AppVeyor and TravisCI will be used to confirm the Biopython unit
tests and style checks pass with these changes.

- [X] I have added my name to the alphabetical contributors listings in the files
``NEWS.rst`` and ``CONTRIB.rst`` as part of this pull request, am listed
already, or do not wish to be listed. (*This acknowledgement is optional.*)

This PR applies black style to test_m* files, files listed below, it would be good to review it.
test_Mafft_tool.py
test_MafIO_index.py
test_MarkovModel.py
test_Medline.py
test_mmtf.py
test_mmtf_online.py
test_motifs.py
test_motifs_online.py
test_MSAProbs_tool.py
test_Muscle_tool.py

### CODE REVIEW

File: Tests/test_MafIO_index.py [hunk 1]
```diff
@@ -302,19 +365,24 @@ def test_correct_retrieval_1(self):
                     3014842, 1371, 7842, 171548, 16169512, 16389874, 15871306, 6404, 184317, 14750994,
                     3018161, 16390178, 15871611, 16169818,
                     3018230, 15871676, 16390243,
-                    3018359, 16390338, 15871771, 184712, 16169976, 3018482})
+                    3018359, 16390338, 15871771, 184712, 16169976, 3018482
+                }
+            )
+            # Turn black code style on
+            # fmt: on
```
Comment (peterjc) [inline Tests/test_MafIO_index.py]: You could try adding comment lines as break points instead, but black would still want to use one line per value.

So what you've done is probably best (turning off black here).

File: Tests/test_MarkovModel.py [hunk 2]
```diff
@@ -281,54 +283,59 @@ def test_save_and_load(self):
...
+                around(markov_model.p_transition, decimals=3),
+                around(output_p_transition, decimals=3),
+            )
+        )
+        self.assertTrue(
+            array_equal(
+                around(markov_model.p_emission, decimals=3),
+                around(output_p_emission, decimals=3),
+            )
+        )
```
Comment (peterjc) [inline Tests/test_MarkovModel.py]: There is probably a numpy way to test two arrays are approximately equal... but that's a side issue.

File: Tests/test_Muscle_tool.py [hunk 3]
```diff
@@ -103,10 +107,13 @@ def test_Muscle_with_options(self):
...
-                         " -in Fasta/f002" +
-                         " -out Fasta/temp_align_out2.fa" +
-                         " -objscore sp -noanchors")
+        self.assertEqual(
+            str(cmdline),
+            _escape_filename(muscle_exe)
+            + " -in Fasta/f002"
+            + " -out Fasta/temp_align_out2.fa"
+            + " -objscore sp -noanchors",
+        )
```
Comment (peterjc) [inline Tests/test_Muscle_tool.py]: Those three strings can be merged, might just fit on one line?

File: Tests/test_Muscle_tool.py [hunk 4]
```diff
@@ -120,9 +127,12 @@ def test_Muscle_profile_simple(self):
...
         cmdline.set_parameter("in2", self.infile3)
-        self.assertEqual(str(cmdline), _escape_filename(muscle_exe) +
-                         " -out Fasta/temp_align_out3.fa" +
-                         " -profile -in1 Fasta/fa01 -in2 Fasta/f001")
+        self.assertEqual(
+            str(cmdline),
+            _escape_filename(muscle_exe)
+            + " -out Fasta/temp_align_out3.fa"
+            + " -profile -in1 Fasta/fa01 -in2 Fasta/f001",
+        )
```
Comment (peterjc) [inline Tests/test_Muscle_tool.py]: Will those two strings fit on one line now?

File: Tests/test_Muscle_tool.py [hunk 5]
```diff
@@ -132,14 +142,22 @@ def test_Muscle_profile_simple(self):
...
+            stable=True,
+            cluster1="neighborjoining",
+        )
+        self.assertEqual(
+            str(cmdline),
+            _escape_filename(muscle_exe)
+            + " -out Fasta/temp_align_out4.fa"
+            + " -profile -in1 Fasta/fa01 -in2 Fasta/f001"
+            + " -cluster1 neighborjoining -stable",
+        )
```
Comment (peterjc) [inline Tests/test_Muscle_tool.py]: Again, can you merge those strings into one or two lines line?

File: Tests/test_Muscle_tool.py [hunk 6]
```diff
@@ -356,18 +390,27 @@ def test_with_multiple_output_formats(self):
...
+            clw=True,
+            htmlout=output_html,
+            clwstrictout=output_clwstrict,
+        )
+        self.assertEqual(
+            str(cmdline).rstrip(),
+            _escape_filename(muscle_exe)
+            + " -in Fasta/f002 -clw -htmlout temp_f002.html"
+            + " -clwstrictout temp_f002.clw",
+        )
```
Comment (peterjc) [inline Tests/test_Muscle_tool.py]: Again, a one line string should work here.

**Comment by peterjc** [comment]
A few multiline strings, but otherwise looks good to me.

File: Tests/test_Muscle_tool.py (hunk 3 above)
Comment (svalqui) [reply Tests/test_Muscle_tool.py]: Done.

File: Tests/test_Muscle_tool.py (hunk 4 above)
Comment (svalqui) [reply Tests/test_Muscle_tool.py]: Done.

File: Tests/test_Muscle_tool.py (hunk 5 above)
Comment (svalqui) [reply Tests/test_Muscle_tool.py]: Done.

File: Tests/test_Muscle_tool.py (hunk 6 above)
Comment (svalqui) [reply Tests/test_Muscle_tool.py]: The coma doesn't fit :(

**Comment by svalqui** [comment]
Amended; Thank you.

File: Tests/test_Muscle_tool.py (hunk 6 above)
Comment (peterjc) [reply Tests/test_Muscle_tool.py]: Ah well, thanks for checking.

**Comment by peterjc** [comment]
Let's give @mdehoon a few days time to review this.

**Comment by peterjc** [comment]
Again, something seems incomplete from the merge... maybe re-run black on these files?

**Comment by peterjc** [comment]
There are still things not following the black style. Can you try re-running black on theses files after the merge?

**Comment by peterjc** [comment]
Rebasing which will hopefully also trigger TravisCI properly this time...

**Comment by peterjc** [comment]
Oops, I'll fix this:

```
$ flake8
./Tests/test_Mafft_tool.py:64:45: ISC001 implicitly concatenated string literals on one line
```

**Comment by peterjc** [comment]
Applied after redoing the git rebase, thank you.

**Pull request merged**